samples/sqlservices/src/controllers/mainController.ts

```diff
@@ -354,15 +355,32 @@ export default class MainController implements vscode.Disposable {
 				webview2.message = count;
 			});
 
-			let flexModel = view.modelBuilder.flexContainer()
-				.withLayout({
-					flexFlow: 'column',
-					alignItems: 'flex-start',
-					height: 500
-				}).withItems([
-					webview1, webview2
-				], { flex: '1 1 50%' })
+			let editor1 = view.modelBuilder.editor()
+				.withProperties({
+					content: 'select * from sys.tables'
+				})
+				.component();
+
+			let editor2 = view.modelBuilder.editor()
+				.withProperties({
+					content: 'print("Hello World !")',
```

Comment:
how about a more generic default for content here?

Reply:
It's the extension's sample code